src/fetch_tse_data.py

```diff
@@ -0,0 +1,254 @@
+import shutil
+import pandas as pd
+import numpy as np
+import os
+import urllib
+import zipfile
+import glob
+
+
+
+FILENAME_PREFIX='consulta_cand_'
+TEMP_PATH = '../data/tse_temp'
+TSE_CANDIDATES_URL='http://agencia.tse.jus.br/estatistica/sead/odsele/consulta_cand/'
+OUTPUT_DATASET_PATH = '../data/2017-03-31-tse-candidates.xz'
+os.makedirs(TEMP_PATH)
+
+# setting year range from 2004 to 2016. this will be modified further to 'from 1994 to 2016'
+year_list=[str(year) for year in (range(2004,2017,2))]
+
+# Download files
+for year in year_list:
+    filename=FILENAME_PREFIX+year+'.zip'
+    file_url=TSE_CANDIDATES_URL+filename
+    output_file=os.path.join(TEMP_PATH,filename)
```

Comment:
Some Python best practices are to use spaces between the object being defined and its value, and also after comas (e.g. `output_file = os.path.join(TEMP_PATH, filename)` instead of `output_file=os.path.join(TEMP_PATH,filename)`). Check [PEP8](http://pep8.readthedocs.io/) or [`prospector`](https://pypi.python.org/pypi/prospector/) if you are interested in this hints on code quality ; )